runtime/v2/runc/manager/manager_linux.go

```diff
@@ -268,3 +276,62 @@ func (manager) Stop(ctx context.Context, id string) (shim.StopStatus, error) {
 		Pid:        pid,
 	}, nil
 }
+
+func (m manager) Info(ctx context.Context, optionsR io.Reader) (*task.RuntimeInfo, error) {
+	info := &task.RuntimeInfo{
+		Name: m.name,
+		Version: &task.RuntimeVersion{
+			Version:  version.Version,
+			Revision: version.Version,
+		},
+		Annotations: nil,
+	}
+	opts, err := shim.ReadRuntimeOptions[*options.Options](optionsR)
+	if err != nil {
+		if !errors.Is(err, errdefs.ErrNotFound) {
+			return nil, fmt.Errorf("failed to read runtime options (*options.Options): %w", err)
+		}
+	}
+	if opts != nil {
+		info.Options, err = protobuf.MarshalAnyToProto(opts)
+		if err != nil {
+			return nil, fmt.Errorf("failed to marshal %T: %w", opts, err)
+		}
+	}
+	features, err := m.features(ctx, opts)
+	if err != nil {
+		// crun and youki do not implement `runc features` yet, at the time of writing this (May 2023)
+		// https://github.com/containers/crun/issues/1177
+		// https://github.com/containers/youki/issues/815
+		log.G(ctx).WithError(err).Debug("Failed to get the runtime features. The runc binary does not implement `runc features` command?")
+	}
+	if features != nil {
+		info.Features, err = protobuf.MarshalAnyToProto(features)
+		if err != nil {
+			return nil, fmt.Errorf("failed to marshal %T: %w", features, err)
+		}
+	}
+	return info, nil
+}
+
+func (m manager) features(ctx context.Context, opts *options.Options) (*features.Features, error) {
+	binaryName := runcC.DefaultCommand
+	if opts != nil && opts.BinaryName != "" {
+		binaryName = opts.BinaryName
+	}
+	absBinary, err := exec.LookPath(binaryName)
```

Comment:
Is it possible to add a Features method to go-runc rather than calling the runc binary directly here?

This could have some similarity to
https://github.com/containerd/containerd/blob/8c21759d7c00acdd81b54ba1007be8022467897b/runtime/v2/runc/manager/manager_linux.go#L252-L256

Reply:
Not necessary IMHO